### PR TITLE
Meta: Drop using a script in the Safari Xcode project

### PR DESCRIPTION
**Pull request by sindresorhus**

### CODE REVIEW

File: safari/Refined GitHub.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -231,13 +276,30 @@
...
+		E3919AC127E6107E009C4956 /* Resources */ = {
 			isa = PBXGroup;
 			children = (
+				E3919AEA27E61192009C4956 /* background.js */,
+				E3919AF327E61192009C4956 /* background.js.map */,
+				E3919AEB27E61192009C4956 /* browser-polyfill.js */,
+				E3919AED27E61192009C4956 /* icon.png */,
+				E3919AE727E61192009C4956 /* manifest.json */,
+				E3919AF627E61192009C4956 /* options.css */,
+				E3919AF527E61192009C4956 /* options.css.map */,
```
Comment (sindresorhus) [inline safari/Refined GitHub.xcodeproj/project.pbxproj:288]: @fregante Should the source map files be included?

Comment (fregante) [reply safari/Refined GitHub.xcodeproj/project.pbxproj:288]: I'll test this later and let you know. I forgot what we decided

Comment (fregante) [reply safari/Refined GitHub.xcodeproj/project.pbxproj:288]: It looks like we include them:

Screen Shot

**Comment by fregante** [comment]
I can't test it because the app crashes on open 😂 probably unrelated but… I cleared `~/Library/Developer/Xcode/DerivedData` too for a clean build and closed Safari. Nothing.

Anyway lgtm and can be merged

**Review by fregante — APPROVED** [review]

**Pull request merged**